src/pages/Lend/LoanSearchPage.vue

```diff
@@ -254,6 +254,11 @@ export default {
 				this.challengeData = goalsData?.goals?.values?.[0] || {};
 				const teamData = this.apollo.readQuery({ query: TeamInfoFromId, variables: { team_id: teamId } });
 				this.teamData = teamData?.community?.team || {};
+
+				const lenderName = this.$route?.query?.lenderName ?? '';
+				if (lenderName && teamId) {
+					this.$showTipMsg(`Support ${lenderName} and help ${this.teamData?.name} hit their goal.`);
```

Comment:
The mocks don't show a tip message, but an element in the page, like we did for the IWD pages.